### PR TITLE
Fix quick start examples

### PR DESCRIPTION
**Pull request by xoscar**
This PR fixes the examples for:

quick-start-net-core
quick-start-nodejs
quick-start-nodejs-manual-instrumentation
quick-start-opensearch-nodejs
quick-start-python
quick-start-tempo-nodejs

To use the new configuration and provisioning methods

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [] updated the docs
- [x] added a test

### CODE REVIEW

File: examples/quick-start-python/tracetest/docker-compose.yaml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
   tracetest:
     image: kubeshop/tracetest:latest
     platform: linux/amd64
     volumes:
-      - ./tracetest/tracetest.config.yaml:/app/config.yaml
+      - type: bind
+        source: ./tracetest/tracetest.config.yaml
+        target: /app/tracetest.yaml
+      - type: bind
+        source: ./tracetest/tracetest.provision.yaml
```
Comment (mathnogueira) [inline examples/quick-start-python/tracetest/docker-compose.yaml:11]: shouldn't we be using `tracetest-provision.yaml`?

**Review by mathnogueira — CHANGES_REQUESTED** [review]
we defined we should use `tracetest-provision.yaml` instead of `tracetest.provision.yaml`

File: examples/quick-start-net-core/tracetest.config.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (schoren) [inline examples/quick-start-net-core/tracetest.config.yaml:1]: the rest of the examples are not using the initial `---`. I would keep the same standard just for consistency

Comment (schoren) [reply examples/quick-start-net-core/tracetest.config.yaml:1]: Also, the rest of the examples are using `tracetest-config.yaml` and `tracetest-provision.yaml` for the file names

**Review by schoren — CHANGES_REQUESTED** [review]
looks good, except for the 2 comments regarding consistency with the rest of the examples

**Comment by xoscar** [comment]
@schoren @mathnogueira please a take a look it should be updated with your requests

**Review by schoren — APPROVED** [review]

**Review by mathnogueira — APPROVED** [review]
LGTM

**Pull request merged**